docs/book/src/tasks/change-machine-template.md

```diff
@@ -0,0 +1,37 @@
+# Changing Infrastructure Machine Templates
+
+Several different components of Cluster API leverage _infrastructure machine templates_, 
+including `KubeadmControlPlane`, `MachineDeployment`, and `MachineSet`. These 
+`MachineTemplate` resources should be immutable, unless the infrastructure provider 
+documentation indicates otherwise for certain fields (see below for more details).
+
+The correct process for modifying an infrastructure machine template is as follows:
+
+1. Duplicate an existing template.
+    Users can use `kubectl get <MachineTemplateType> <name> -o yaml > file.yaml` 
```

Comment:
Not sure if you're trying to have this formatted as-written in here, but the whitespace & indentation are lost when rendered. See https://deploy-preview-3105--kubernetes-sigs-cluster-api.netlify.app/tasks/change-machine-template.html.

Reply:
I'm OK with the rendered version if you are. That wasn't _quite_ what I was shooting for, but it's acceptable IMO.